src/oas/__tests__/__snapshots__/operation.test.ts.snap

```diff
@@ -337,13 +337,13 @@ Array [
               "minimum": -9223372036854776000,
               "type": "number",
               "x-stoplight": Object {
-                "id": "3c572d09f18de",
+                "id": "8a951cfda4990",
               },
             },
             "style": "simple",
           },
         ],
-        "id": "ae2d0e59a4075",
+        "id": "ee11fabc87ff7",
```

Comment:
this one is due to `produces` being empty
Before the change we did

```ts
hash(`http_response-${this.parentId}-${actualKey}-${sortProducesOrConsumes(produces).join('-')}`)
```

and right now we do

```ts
hash([
      'http_response',
      props.parentId,
      props.codeOrKey,
      ...(Array.isArray(props.produces) ? sortAlphabetically(props.produces) : []),
]);
```

In the first case, we'll always have the trailing dash, but in the latter scenario that dash won't be there, hence the change in the ID.